packages/components/src/drop-zone/index.tsx

```diff
@@ -126,6 +113,7 @@ export function DropZoneComponent( {
 	const [ isDraggingOverElement, setIsDraggingOverElement ] =
 		useState< boolean >();
 	const [ type, setType ] = useState< DropType >();
+	//const disableMotion = true;
```

Comment:
Opsie, I just saw that. It's a leftover from a previous test I made. I will remove it.